config.go

```diff
@@ -118,6 +128,18 @@ func loadConfig() Config {
 		log.Fatalf("error while unmarshalling config: %v", err)
 	}
 
+	// Load the API key
```

Comment:
I think `nutsnodeapiuser` is required now, so can lead to an error when nog configured